src/lib/y2storage/clients/inst_disk_proposal.rb

```diff
@@ -110,6 +107,9 @@ def expert_partitioner
         when :next
           @proposal = nil
           @devicegraph = dialog.device_graph
+        when :back
+          # Try to create a proposal when staging has not changed (i.e., after rescannig devices)
```

Comment:
I find the wording of the comment a little bit confusing. I would say "when staging has been reseted to probed". Or "when staging contains a copy of probed", or something like that.